### DISCUSSION TITLE
USB Webcam ussage as live view and capture

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#206] [Q&A] [answered]
I'm not shure but when I've installed the software. It was possible to use the USB webcam for both backends.
After I managed to use my Nikon D3100 for Capture, I set both backends to webcamcv2.

Now I have the following behavior.
After reboot livestream is not working. Capture is working. un- and replug the webcam, Liveview works but no picture captured.

Is it right, that it should be possibe to use one USB webcam for both backends ?

**Comment by mgineer85** [comment]
Use just one backend, never two times the same. It will access you camera twice which will result in errors

**Reply by Striker62** [reply, marked as answer]
ok then I used at the beginning the webcam and the raspicam
Thanks
